.github/workflows/tests.yml

```diff
@@ -1,5 +1,5 @@
 name: tests
-on: [pull_request, workflow_dispatch]
+on: [pull_request]
```

Comment:
Was there a  particular reason to disable it? It seemed useful to be able to trigger it from one's own fork during the development of a new thing, without having to spam a PR with a dozen pushes

Reply:
Right now it takes the pull number from `GITHUB_REF`, as in this really only works right now with PRs against main.